drivers/spi/Kconfig.nrfx

```diff
@@ -14,6 +14,20 @@ menuconfig SPI_NRFX
 
 if SPI_NRFX
 
+if SOC_NRF52832
+
+config SOC_NRF52832_PAN_58_OVERRIDE
+	bool "Allow enabling the SPIM driver despite PAN 58"
+	help
+		Allow enabling the nRF SPI Master with EasyDMA, despite
+		Product Anomaly Notice 58 (SPIM: An additional byte is
+		clocked out when RXD.MAXCNT = 1). Without this
```

Comment:
What about stating it also explicitly: "Use this option only if you are certain that affected transactions will not be executed"?

Reply:
Heh. Because it's too easy to write things like that and forget to make a clear, conclusive statement.

Nicely worded, thanks.

Fixed.